app/src/pages/inside/membersPage/modals/permissionMapModal/permissionMap/permissionMap.jsx

```diff
@@ -0,0 +1,387 @@
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import styles from './permissionMap.scss';
+import { Permissions } from './permissions';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class PermissionMap extends Component {
+  static propTypes = {
+    intl: intlShape,
+  };
+  static defaultProps = {
+    intl: {},
+  };
+  okClickHandler = (closeModal) => {
+    setTimeout(() => closeModal(), 1000);
+  };
+  render() {
+    const { intl } = this.props;
+    return (
+      <div className={cx('container')}>
```

Comment:
Can we somehow automate permission map generation? I mean, it would be nice to have some object containing permissions, and this component could generate markup based on this definition.
This will make the component easier to read and maintain.